setup.py

```diff
@@ -26,13 +26,13 @@
 dependecy_links = []
 if sys.platform == "win32":
     if sys.version_info[:2] == (3, 5):
-        DEPENDENCY_LINKS = ["https://download.pytorch.org/whl/cpu/torch-1.5.0%2Bcpu-cp35-cp35m-win_amd64.whl"]
+        dependecy_links = ["https://download.pytorch.org/whl/cpu/torch-1.5.0%2Bcpu-cp35-cp35m-win_amd64.whl"]
```

Comment:
https://stackoverflow.com/questions/12518499/pip-ignores-dependency-links-in-setup-py